pandas/_libs/tslibs/strptime.pyx

```diff
@@ -396,7 +396,8 @@ def array_strptime(
 
             result_timezone[i] = tz
 
-        except (ValueError, OutOfBoundsDatetime):
+        except (ValueError, OutOfBoundsDatetime) as ex:
+            ex.args = (str(ex) + f" at position {i}", )
```

Comment:
similar to

https://github.com/pandas-dev/pandas/blob/2f54a474aa754f37f82dab0184570e421207ba6d/pandas/_libs/tslib.pyx#L789-L790

I just removed the word `'present'` as I don't think sounds good with the errors from this file

i.e.
```
time data ' ' does not match format '%m/%d/%Y' \(match\) at position 2
```
sounds better than
```
time data ' ' does not match format '%m/%d/%Y' \(match\) present at position 2
```

Reply:
Nit but why not use a full f-string instead of addition between 2 strings here?